src/components/TransactionsTable/components/TransactionRow.tsx

```diff
@@ -131,7 +137,9 @@ export const TransactionRow = ({
         <td>
           <div className='d-flex'>
             <span
-              className={`badge badge-outline badge-rounded badge-direction ${direction.toLowerCase()}`}
+              className={`badge badge-outline badge-rounded badge-direction ${direction
```

Comment:
This could leverage on the usage of `classNames`, and the processed `direction` string should be stored in variable and that should be called instead. 👍